### PR TITLE
Fill MsgBase

### PR DESCRIPTION
**Pull request by smellthemoon**
Signed-off-by: xinguo.li <xinguo.li@zilliz.com>
#19639

### CODE REVIEW

**Comment by smellthemoon** [comment]
rerun ut

File: internal/util/commonpbutil/commonpbutil.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+func NewMsgBase(options ...MsgBaseOptions) *commonpb.MsgBase {
+	msgBase := NewMsgBaseDefault()
+	for _, op := range options {
+		op(msgBase)
+	}
+	return msgBase
+}
+
+func NewMsgBaseCopy(msgBase *commonpb.MsgBase, options ...MsgBaseOptions) *commonpb.MsgBase {
+	msgBaseRt := msgBase
```
Comment (congqixia) [inline internal/util/commonpbutil/commonpbutil.go:86]: NIT: do we need clone here?

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**